frontend/app/routes/$lang/_public/apply/$id/child/children/$childId/information.tsx

```diff
@@ -217,58 +216,26 @@ export default function ApplyFlowChildInformation() {
   const params = useParams();
   const fetcher = useFetcher<typeof action>();
   const isSubmitting = fetcher.state !== 'idle';
-  const errorSummaryId = 'error-summary';
   const [hasSocialInsuranceNumberValue, setHasSocialInsuranceNumberValue] = useState(defaultState?.hasSocialInsuranceNumber);
 
+  const errors = fetcher.data?.errors;
+  const errorSummary = useErrorSummary(errors, {
+    firstName: 'first-name',
+    lastName: 'last-name',
+    ...(i18n.language === 'fr'
+      ? { dateOfBirth: 'date-picker-date-of-birth-day', dateOfBirthDay: 'date-picker-date-of-birth-day', dateOfBirthMonth: 'date-picker-date-of-birth-month' }
+      : { dateOfBirth: 'date-picker-date-of-birth-month', dateOfBirthMonth: 'date-picker-date-of-birth-month', dateOfBirthDay: 'date-picker-date-of-birth-day' }),
+
+    dateOfBirthYear: 'date-picker-date-of-birth-year',
+    socialInsuranceNumber: 'social-insurance-number',
+    hasSocialInsuranceNumber: 'input-radios-has-social-insurance-number',
+    isParent: 'input-radios-is-parent-radios',
```

Comment:
I noticed that for radio button errors, the error would link to the first radio option, but these two errors link to the fieldset. Is that intentional? Feel free to resolve since this is how it currently behaves.

Reply:
It's something we have to look at. It's been there before a made my changes.